src/lib/core/overlay/overlay-directives.ts

```diff
@@ -50,15 +52,51 @@ export class OverlayOrigin {
  * Directive to facilitate declarative creation of an Overlay using a ConnectedPositionStrategy.
  */
 @Directive({
-  selector: '[connected-overlay]'
+  selector: '[connected-overlay]',
+  exportAs: 'connectedOverlay'
 })
-export class ConnectedOverlayDirective implements OnInit, OnDestroy {
+export class ConnectedOverlayDirective implements OnDestroy {
   private _overlayRef: OverlayRef;
   private _templatePortal: TemplatePortal;
+  private _open = false;
+  private _hasBackdrop = false;
+  private _backdropSubscription: Subscription;
 
   @Input() origin: OverlayOrigin;
   @Input() positions: ConnectionPositionPair[];
 
+  /** The width of the overlay panel. */
+  @Input() width: number | string;
+
+  /** The height of the overlay panel. */
+  @Input() height: number | string;
+
+  /** The custom class to be set on the backdrop element. */
+  @Input() backdropClass: string;
+
+  /** Whether or not the overlay should attach a backdrop. */
+  @Input()
+  get hasBackdrop() {
+    return this._hasBackdrop;
+  }
+
+  set hasBackdrop(value: any) {
+    this._hasBackdrop = value === '' || (!!value && value !== 'false');
```

Comment:
Should this be the same logic as here?
https://github.com/jelbourn/material2/blob/a70e3e569d026aaf4c0ad8728d9b6091f8673901/src/lib/core/coersion/boolean-property.ts#L3